eCommerce/users/serializers.py

```
@@ -0,0 +1,133 @@
+    class Meta:
+        model = Address
+        fields = '__all__'
+
+class UserAddressSerializer(serializers.ModelSerializer):
+    address = AddressSerializer()
+
+    class Meta:
+        model = UserAddress
+        fields = ('id', 'address', 'is_default')
+        # Add ordering to fix pagination warning
+        ordering = ['-is_default']
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)